packages/mobile/src/screens/favorites-screen/FavoritesDownloadSection.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react'
+
+import { View } from 'react-native'
+
+import { makeStyles } from 'app/styles'
+
+import { DownloadFavoritesSwitch } from './DownloadFavoritesSwitch'
+import { DownloadProgress } from './DownloadProgress'
+import { FavoritesDownloadStatusIndicator } from './FavoritesDownloadStatusIndicator'
+
+const useStyles = makeStyles(() => ({
+  root: { flexDirection: 'row', alignItems: 'center' }
+}))
+
+export const FavoritesDownloadSection = () => {
```

Comment:
Awesome job breaking the component into smaller pieces! I wonder if there's a more descriptive name for this one than `Section`, I can't think of one though lol

Reply:
Me too! I tried for a while and then just went with this... like header maybe?